.github/workflows/ci-actions.yml

```diff
@@ -1,8 +1,29 @@
 name: Jakarta Contexts and Dependency Injection CI
 
 on:
+  push:
```

Comment:
I think the idea here is that all changes go in via PRs so this will only really trigger during a release (since the bot does direct push) and that's kind of pointless. At least from my perspective, your mileage may vary :)

Reply:
I activated triggering the ci on a push on a custom branch for a given push of one or many commits, thing that changes from the previous configuration is that now I can see the build status on my own branch before merging it, this will someway prevent me to make a pull request with changes that can break the build or the tests 


<img width="1505" alt="image" src="https://github.com/jakartaee/cdi/assets/45496/698ee0d5-c92e-48fd-b985-412007d45dd5">